www/main/navigator/misc/components/MiscHeader.jsx

```diff
@@ -35,6 +36,16 @@ export const MiscHeader = () => {
           </span>
         </i>
       </a>
+      <a
+        className={`misc-button ${isFav ? 'misc-active' : ''}`}
```

Comment:
there is a utility function in application named `classNames` (custom function) you can refactor this code something like this:

```jsx
className={classNames('misc-button', isFav && 'misc-active')}
```

Just in case the name of this function is something else I'm enclosing the function body as well

```js
const classNames = (...classes) => classes.filter(Boolean).join(' ');
```